cmd/buildah/push.go

```diff
@@ -179,3 +218,28 @@ func getListOfTransports() string {
 	allTransports := strings.Join(transports.ListNames(), ",")
 	return strings.Replace(allTransports, ",tarball", "", 1)
 }
+
+// getAllTags gets all of the locally tagged images and returns them
+// in a slice.
+func getAllTags(image string, imageTags map[string]string, store storage.Store) error {
+
+	images, err := store.Images()
+	if err != nil {
+		return errors.Wrapf(err, "error reading images")
+	}
+	for _, image := range images {
+
+		if len(image.Names) < 1 {
+			continue
+		}
+
+		for _, name := range image.Names {
+			splitName := strings.Split(name, ":")
+			if len(splitName) > 1 {
+				imageTags[splitName[1]] = ""
```

Comment:
Why use a map rather then a list?

Reply:
'cause I love Maps?  Actually thought I'd use it as a quick lookup later, but you're right I ended up iterating so I'll pop it back to a slice.